apps/aevm/src/aevm_data.erl

```diff
@@ -211,7 +216,11 @@ convert(Input, Output, MaxSize, Store, Visited, typerep, Ptr, Heap, BaseAddr) ->
                          [{list, typerep}],          %% tuple
                          [{list, {list, typerep}}],  %% variant
                          [],                         %% typerep
-                         [typerep, typerep]          %% map
+                         [typerep, typerep],         %% map
+                         [not_a_valid_type_crash],   %% fun - not allowed as argument, should crash, only
```

Comment:
🤔

Reply:
I think we could get away without this at the end of the day...